utils/s3/client.cc

```diff
@@ -696,7 +708,9 @@ future<> client::multipart_upload::abort_upload() {
 
 future<> client::multipart_upload::finalize_upload() {
     s3l.trace("wait for {} parts to complete (upload id {})", _part_etags.size(), _upload_id);
-    co_await _bg_flushes.close();
+    if (!_bg_flushes.is_closed()) {
```

Comment:
How can this happen?

Reply:
Addressed [here](https://github.com/scylladb/scylladb/pull/20518/files#diff-28c1d03a5facd99f25f466e6bae94c0fab2bb8845c0538dd489db7832505e884R785)